### DISCUSSION TITLE
Matter-Server for Homeassistant

### DISCUSSION THREAD

**Discussion by MichelFR** [tteck/Proxmox#816] [Ideas]
As stated here: https://www.home-assistant.io/integrations/matter#advanced-installation-instructions
It is needed to host a custom matter server for homeassistant.
Would be nice to ahve a install script for that

**Comment by tteck** [comment]
Looking into

**Reply by tteck** [reply]
When needed, we will have a Matter-Server LXC

**Comment by TychoWerner** [comment]
It's part of HA now it seems

**Reply by tteck** [reply]
Not if your running Core or Container. You need a Matter server (which I have ready for testing... soon)

**Reply by TychoWerner** [reply]
Ah fair point!

**Reply by GyHUN95** [reply]
Is there any update about it or anything we can help with?

**Reply by tteck** [reply]
Things are changing so fast, I've been waiting to see where it goes.
https://github.com/home-assistant-libs/python-matter-server
